packages/@okta/vuepress-site/docs/guides/submit-app/main/index.md

```diff
@@ -60,13 +60,48 @@ If the Okta OIN team identifies any issues in the initial review and QA testing
 
   >**Note:** SWA app integrations are no longer accepted for publication in the OIN catalog. However, existing SWA apps are still maintained by the OIN team.
 
+### Understand the submission review process
+
+The submission review process begins after you click **Submit for Review** in the [OIN Manager](https://oinmanager.okta.com). Okta sends you an email notification that your integration is now queued for review by the Okta OIN team, along with a date by which you should complete the initial review.
+
+The OIN Manager always shows the current status of your integration.
```

Comment:
Is 'always' necessary? 
Your watch shows the current time.
Your watch always shows the current time.